proto/acl_counts.proto

```diff
@@ -0,0 +1,29 @@
+/*
+ * ACL counts for devices
+ */
+syntax = "proto3";
+
+/*
+ * ACL counts for devices
+ */
+message AclCounts {
+  // List of ACL counts indexed by device MAC addresses
+  map<string, AclCount> devices = 1;
+}
+
+/*
+ *  ACL counts for a device
+ */
+message AclCount {
```

Comment:
I think this should be RuleCounts. AclCounts == map -> AclCount, RuleCounts --> map -> RuleCount

Reply:
So something like this?

```
message AclCounts {
  // List of ACL counts indexed by device MAC addresses
  map<string, RuleCounts> devices = 1;     <--------------------- changed from AclCount to RuleCounts
}

message RuleCounts {            <--------------------------- changed from AclCount to RuleCounts
  // List of rule counts indexed by the descriptions of the rules
  map<string, RuleCount> rules = 1;
  repeated string errors = 2;
}
```